spinn_front_end_common/interface/java_caller.py

```diff
@@ -219,12 +184,12 @@ def set_report_folder(self, report_folder):
         """
         self._report_folder = report_folder
 
-    def set_placements(self, placements, transceiver):
+    def set_placements(self, used_placements):
         """ Passes in the placements leaving this class to decide pass it to
             Java.
 
         This method may obtain extra information about he placements which is
-        why it also needs the transceiver.
+        why it also accesses the transceiver.
```

Comment:
I guess it does access the transceiver through later functions, but this function itself doesn't actually access it, so perhaps this whole comment is unnecessary.

Reply:
confusing comment removed.